### DISCUSSION TITLE
which plugin might have generated the wavy underline with no diagnostics?

### DISCUSSION THREAD

**Discussion by ssh352** [LazyVim/LazyVim#3162] [Q&A]
<img width="2548" alt="Screenshot 2024-05-14 at 11 56 09 PM" src="https://github.com/mrcjkb/rustaceanvim/assets/41779116/14002e25-5ed2-4ddc-b224-d0fd0fc9301a">

https://github.com/mrcjkb/rustaceanvim/discussions/390

Initially I thought it is caused by rustacean.vim. Also the lazyvim rust.lua works perfectly. Just not sure what caused the underlines.

And I've tried configure nvim-lspconfig
```
  opts = {
              diagnostics = { underline = false },
}
```
to no avail.

thanks!

**Comment by dusty-phillips** [comment]
possible spelling. If you do `<Space>us` do they disappear? I'm not sure which plugin enables spell checking of if it's a built-in setting.

**Reply by ssh352** [reply]
No it is still there. I am going to disable plugin one by one to find out. Thanks!

**Reply by ssh352** [reply]
for the record, it is the tokyonight color scheme. other color schemes have no such issue.
